torch/csrc/jit/passes/quantization.cpp

```diff
@@ -948,6 +948,8 @@ graph(%self, %scale, %zero_point, %dtype):
         zero_point_node->kind() == prim::Constant &&
         dtype_node->kind() == prim::Constant;
   };
+  std::unordered_set<Node*> quantize_nodes_to_delete;
```

Comment:
Can you combine these two sets?

Reply:
I can just use a vector to preserve the topological ordering

Reply:
Actually if I just reuse the `GetAttr` node we can just avoid deleting those altogether